### ISSUE TITLE
Pangolin doesn't update as expected

### ISSUE THREAD

**Issue by aegaskin** [cov-lineages/pangolin#386]
Hello, I was wondering if I could raise an issue I've been experiencing with Pangolin. (If there's an easy solution to this, please let me know, because that'd be amazing!) 😄 

I've been logging the version of pangolin I use by first running something along the lines of:

**Check the version already installed**
`pangolin --all-versions`

**Update pangolin**
`pangolin --update`

**Check the version now installed - has it changed?**
`pangolin --all-versions`

Recently, when I have been running `pangolin --update`, it updates `pango-designation updated to v1.2.134`, evidenced by how it prints this to screen. However, when I run `pangolin --all-versions` before and after, `pango-designation aliases: 1.2.133` remains constant. Is this behaviour expected?

Thanks in advance, and sorry if this ends up being a basic thing or if I've collided with people working on code!

**Comment by SharifShaaban-PHS** [comment]
Hi, I am seeing the same issue, however, would like to add that further to the behaviour above, `pango-designation used by pangoLEARN/Usher: v1.2.127` also remains constant and does not update.

**Comment by Penpitchaisaluckygirl** [comment]
Hi there, I've got the same problem. 
I updated using pangolin --update, and I think the program meet the last version.
 
pangolin already latest release (v3.1.20)
pangolearn already latest release (2022-02-28)
constellations already latest release (v0.1.4)
scorpio already latest release (v0.3.16)
pango-designation updated to v1.2.134

After runing, pangolin lineage didn't update.

**Comment by aineniamh** [comment]
Thanks for flagging, I'll look into this as soon as possible (currently on annual leave, but may get a chance tomorrow)!

**Comment by aineniamh** [comment]
Okay I've looked at this using these versions
```
pangolin: 3.1.20
pangolearn: 2022-02-28
constellations: v0.1.4
scorpio: 0.3.16
pango-designation used by pangoLEARN/Usher: v1.2.127
pango-designation aliases: 1.2.133
```
Everything bar the pango-designation aliases is the latest version released (@SharifShaaban-PHS your issue is resolved because that's the latest version available, that can be checked on the pangolearn repo).

I run `pangolin --update`
```
pangolin already latest release (v3.1.20)
pangolearn already latest release (2022-02-28)
constellations already latest release (v0.1.4)
scorpio already latest release (v0.3.16)
pango-designation updated to v1.2.136
```

And then `pangolin --all-versions` gives:
```
pangolin: 3.1.20
pangolearn: 2022-02-28
constellations: v0.1.4
scorpio: 0.3.16
pango-designation used by pangoLEARN/Usher: v1.2.127
pango-designation aliases: 1.2.136
```
I'm finding I can't replicate the issue as this is expected behaviour.

Bear in mind that pango-designation used for alias file is distinct to the pango-designation version that is reported in relation to the trained model.

**Comment by aineniamh** [comment]
The pango_version is the version that the model was trained on, whereas the  pango-designation aliases is just the version of pango-designation installed so that pangolin can print out the alias file. 
In the next few days I'm going to release pangolin 4.0 that won't have pango-designation as a dependency, and will just have a single version that corresponds to the version of the data the model and protobuf file correspond to.

**Comment by indexofire** [comment]
I have the same issue that pangolin was updated by conda. I installed it in a new environment, the designation version still stay in 1.2.124 which will not include some lineages like BA.2.2 ? 

```
pangolin: 3.1.20
pangolearn: 2022-02-02
constellations: v0.1.4
scorpio: 0.3.16
pango-designation used by pangoLEARN/Usher: v1.2.124
pango-designation aliases: 1.2.137
```

**Comment by aegaskin** [comment]
Thanks for taking a look at this, Áine, it's much appreciated!

**Comment by aineniamh** [comment]
So just checking why pangolearn wouldn't have updated @indexofire- the [conda recipe](https://github.com/bioconda/bioconda-recipes/blob/fbf9ba005caa86c6d8b0ad9f209b9926a49bdbe5/recipes/pangolearn/meta.yaml) is not specifying the latest model version, so bioconda only has the second most recent release. 

There'll be a new release today, so it should hopefully all get updated then- @pvanheus @druvus, is this something that can be automated at all? I'm not very familiar with the bioconda process!

**Comment by indexofire** [comment]
@aineniamh I installed all packages via pip+git, the version updated and everything was fine. I also update pangolin to v4.0, the result is correct either. thanks.

**Comment by aineniamh** [comment]
Excellent! Glad it's all working!

**Comment by druvus** [comment]
@aineniamh I have added a PR to update the conda pangoLEARN version (for those that are using pangolin 3). The bioconda bot should detect the updates automatically but it seems to be some issues with pangoLEARN and some other packages. I suspect it might be a bug not handling capital letters correctly but I haven't had to look into it. 

I will try to fix the update of pangolin 4.0 in Bioconda by fixing the dependencies.
